include/plotIt.h

```diff
@@ -57,6 +57,19 @@ namespace plotIt {
       return errors_type;
   }
 
+  enum Log {
```

Comment:
Indeed it should work with an enum class :) But all others are simple enum so it'd break harmony :smiley: